### ISSUE TITLE
Unable to get same hashrate on Linux as Windows

### ISSUE THREAD

**Issue by fractalyse** [fireice-uk/xmr-stak-cpu#327]
Hi,

I'm currently running Gentoo x64 under kernel 4.9.34.
On Windows with xmr-stak-cpu I have around 590h/s with my i7 7820x @ 4.6Ghz, but under Linux I only have around 508h/s.

I tried to modify numa nodes in my kernel, best results are with 6 nodes, 4 or 8 gaves me significant lower results, 280h/s for 4 nodes, and 350h/s for 8.

If I start xmr-stak-amd, wait for 30sec, then I start xmr-stak-cpu, computer become very laggy, totaly unusable, until I kill xmr-stak-cpu, during this time, hasrate drop to  370-400h/s.

I tried to compile it with and without custom CFLAGS, same results.

Thanks a lot for your work.
Cheers

**Comment by fractalyse** [comment]
Solved with new kernel (4.12.12)

**Comment by Dev1436** [comment]
@fractalyse Hello, sorry I know this is a bit off topic, but I haven't been able to google the answer.

How were you able to get this to compile on Gentoo? I keep getting errors.

```
Run Build Command:"/usr/bin/gmake" "cmTC_5847b/fast"
/usr/bin/gmake -f CMakeFiles/cmTC_5847b.dir/build.make CMakeFiles/cmTC_5847b.dir/build
gmake[1]: Entering directory '/home/dev1436/Downloads/MiningOps/Monero/Miners/xmr-stak-cpu-1.3.0-1.5.0/CMakeFiles/CMakeTmp'
Building C object CMakeFiles/cmTC_5847b.dir/CheckFunctionExists.c.o
/usr/bin/cc    -DCHECK_FUNCTION_EXISTS=pthread_create   -o CMakeFiles/cmTC_5847b.dir/CheckFunctionExists.c.o   -c /usr/share/cmake/Modules/CheckFunctionExists.c
Linking C executable cmTC_5847b
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_5847b.dir/link.txt --verbose=1
/usr/bin/cc   -DCHECK_FUNCTION_EXISTS=pthread_create    CMakeFiles/cmTC_5847b.dir/CheckFunctionExists.c.o  -o cmTC_5847b -rdynamic -lpthreads 
/usr/lib/gcc/x86_64-pc-linux-gnu/5.4.0/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -lpthreads
collect2: error: ld returned 1 exit status
gmake[1]: *** [CMakeFiles/cmTC_5847b.dir/build.make:98: cmTC_5847b] Error 1
gmake[1]: Leaving directory '/home/dev1436/Downloads/MiningOps/Monero/Miners/xmr-stak-cpu-1.3.0-1.5.0/CMakeFiles/CMakeTmp'
gmake: *** [Makefile:126: cmTC_5847b/fast] Error 2
```

Thanks for any help
